src/add/operator/let.ts

```diff
@@ -0,0 +1,6 @@
+import {Observable} from '../../Observable';
+import {letProto} from '../../operator/let';
+Observable.prototype.let = letProto;
+Observable.prototype.letBind = letProto;
```

Comment:
This file is auto-generated by a script. In order to generate it properly next time the script is executed you have to:

1. Put the real name of the operator as `let` instead of `letProto` ([generate-operator-patches.ts:40](https://github.com/ReactiveX/RxJS/blob/master/tools/generate-operator-patches.ts#L40))
2. Add `letBind` as an alias ([generate-operator-patches.ts:48](https://github.com/ReactiveX/RxJS/blob/master/tools/generate-operator-patches.ts#L48))